src/interface/storage.thrift

```diff
@@ -334,6 +343,7 @@ struct AddVerticesRequest {
         (cpp.template = "std::unordered_map")   prop_names,
     // if ture, when (vertexID,tagID) already exists, do nothing
     4: bool                                     if_not_exists,
+    5: optional RequestCommon                   common,
```

Comment:
not used?

Reply:
Yes, it is not used. But I think it is better to define uniformly in all Requests